Gems/Terrain/Assets/Materials/Terrain/TerrainDetailMaterial.materialtype

```diff
@@ -0,0 +1,142 @@
+{
+    "description": "Material Type based on standard PBR with additional properites related to terrain.",
+    "version": 1,
+    "propertyLayout": {
+        "propertyGroups": [
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/BaseColorPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/MetallicPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/RoughnessPropertyGroup.json"
+            },
+            {
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/SpecularPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/NormalPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/OcclusionPropertyGroup.json"
+            },
+            { 
+                "$import": "../../../../Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/ParallaxPropertyGroup.json" 
+            },
```

Comment:
This looks a little bit sketchy, as it assumes the location of the Terrain Gem vs the Atom Gem. I won't block the PR over it since both gems are currently distributed together in the engine, but is there a better way to describe the base relative path to look this up from? like "@GEMROOT:Atom@" or something?

Reply:
Actually I think @santorac emailed about this to try to figure out a better solution and we may be able to do use a callback to resolve the path to just Materials/Types/etc... we can do that later though.